classList.go

```diff
@@ -0,0 +1,15 @@
+package dom
+
+import "github.com/dennwc/dom/js"
+
+type ClassList struct {
```

Comment:
According to [docs](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList) it should be `dom.TokenList`.